### PR TITLE
[Samples] Add headings to sample cards

### LINKED ISSUE #7607: [Bug 40020946](https://microsoft.visualstudio.com/OS/_workitems/edit/40020946): [AC UWP Visualizer app sample cards]: Many of the sample cards does not have a 'heading' instead a regular text is present represent the heading for the sample card.
### Problem Description

**Reference:** [Bug 40020946](https://microsoft.visualstudio.com/OS/_workitems/edit/40020946): [AC UWP Visualizer app sample cards]: Many of the sample cards does not have a 'heading' instead a regular text is present represent the heading for the sample card.

### Card JSON

```JSON
**Reference:** [Bug 40020946](https://microsoft.visualstudio.com/OS/_workitems/edit/40020946): [AC UWP Visualizer app sample cards]: Many of the sample cards does not have a 'heading' instead a regular text is present represent the heading for the sample card.
```

### Screenshot

_No response_

### PR DESCRIPTION
**Pull request by anna-dingler**
# Related Issue

Fixes #7607 

# Description

Add headings to sample cards.

I updated and moved required cards to v1.5 so that we could include `TextBlock.style`. Added `TextBlock.heading` to all cards already at v1.5 and above.

# How Verified

Verified on the AdaptiveCards site.

image

# Outstanding issue

Updating to v1.5 exposed a Table rendering issue in UWP - https://github.com/microsoft/AdaptiveCards/issues/8051

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/8045)

### CODE REVIEW

File: source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs [hunk 1]
```diff
@@ -97,12 +97,11 @@ private void TestPayloadsInDirectory(string path, string[] excludedCards)
             }
         }
 
-        [TestMethod]
-        public void TestAllScenarios()
-        {
-            TestPayloadsInDirectory(Path.Combine(SamplesPath, "v1.0", "scenarios"), null);
-            TestPayloadsInDirectory(Path.Combine(SamplesPath, "v1.1", "scenarios"), null);
-        }
+        // [TestMethod]
```
Comment (anna-dingler) [inline source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs:100]: I don't think .NET supports v1.5, so I commented out these tests.

@paulcam206 @almedina-ms is this the correct behavior?

File: source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp [hunk 2]
```diff
@@ -73,19 +73,25 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
 
             double cellSpacingDouble = static_cast<double>(tableConfig.CellSpacing());
 
-            // Set left and top margin for each cell (to avoid double margins). Don't set the margin on topmost
-            // or leftmost cells to avoid creating margin outside the table.
-            winrt::Thickness marginThickness = {cellSpacingDouble, cellSpacingDouble, 0, 0};
-            if (columnNumber == 0)
+            if (cellSpacingDouble != 0)
```
Comment (anna-dingler) [inline source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp:76]: We were crashing here, so I added a check to only update Margin if there is a non-zero value.

File: source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp [hunk 3]
```diff
@@ -73,19 +73,25 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
...
             {
-                marginThickness.Left = 0;
+                // Set left and top margin for each cell (to avoid double margins). Don't set the margin on topmost
+                // or leftmost cells to avoid creating margin outside the table.
+                winrt::Thickness marginThickness = {cellSpacingDouble, cellSpacingDouble, 0, 0};
+                if (columnNumber == 0)
```
Comment (jwoo-msft) [inline source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp:81]: nit: spacing isn't consistent; between if statements; try running clang-format. it should fix it for you.

File: source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs (hunk 1 above)
Comment (jwoo-msft) [reply source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs:100]: v1.5 is code complete for dot net with two PR open.

**Review by jwoo-msft — APPROVED** [review]

File: source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs (hunk 1 above)
Comment (anna-dingler) [reply source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs:101]: I created an issue to address this once .NET is ready for v1.5 - #8071

File: source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp (hunk 3 above)
Comment (anna-dingler) [reply source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp:80]: I ran clang-format and removed some of the empty lines.

**Pull request merged**